Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -137,6 +137,46 @@ public extension UIButton {
             setTitle(newValue, for: .selected)
         }
     }
+    
+    /// SwifterSwift: Attributed title of disabled state for button.
+    var attributedTitleForDisabled: NSAttributedString? {
+        get {
+            return attributedTitle(for: .disabled)
```

Comment:
In modern Swift you don't need the `return` value if there's only 1 line. And you can even shorten it to 1 line, which you could probably do for all of these.
```swift
get { attributedTitle(for: .disabled) }
set { setAttributedTitle(newValue, for: .disabled) }
```
I know it's not necessarily consistent with the rest, but I think it's the clean way to do it.

Reply:
Done.